### ISSUE TITLE
Update App Store screenshots for Big Sur and add them to the readme

### ISSUE THREAD

**Issue by brendanlensink** [steamclock/Quests#1]
Our current screenshots in the app store description are using the old menu bar style. It would also be nice to add them to the readme to give browsers a little bit more of an idea of what they're looking at.

I believe @bradyv has the Figma files for this?

**Comment by bradyv** [comment]
@brendanlensink Yep they're in Figma. We can update the system icons in the mockup, but we also have a version at the ready that only includes the Quests bar item that's a lot simpler and probably just as effective

<img width="1000" alt="quests" src="https://user-images.githubusercontent.com/889872/103556493-b4b00f80-4e66-11eb-930b-531d8338bc94.png">

**Comment by brendanlensink** [comment]
@bradyv I was mostly just hoping to update the screenshots to match the new menu bar, where it's more transparent and coloured based on the background image behind it.
![Screen Shot 2021-01-04 at 9 06 16 AM](https://user-images.githubusercontent.com/3431302/103560127-1aeb6100-4e6c-11eb-87ca-ac070b37426a.png)

**Comment by bradyv** [comment]
Ahh of course! Here's a version with and without the system items (I had to duplicate a kit to get the bar right so they came for free)

<img width="1000" alt="quests b" src="https://user-images.githubusercontent.com/889872/103560900-7538f180-4e6d-11eb-9ecd-484ab5a389b8.png">


<img width="1000" alt="quests a" src="https://user-images.githubusercontent.com/889872/103560893-71a56a80-4e6d-11eb-8e5f-0824d7c1189d.png">

**Comment by brendanlensink** [comment]
Is it possible to do a light mode one as well easily as well @bradyv ?

**Comment by bradyv** [comment]
@brendanlensink Had to remove the loupe because it was causing problems but otherwise super easy to swap between light and dark mode.

## Light
<img width="1000" alt="quests a" src="https://user-images.githubusercontent.com/889872/103562590-30fb2080-4e70-11eb-9901-e978db04d82f.png">
<img width="1000" alt="quests b" src="https://user-images.githubusercontent.com/889872/103562593-32c4e400-4e70-11eb-9e19-a4c58c4c4c8e.png">

## Dark
<img width="1000" alt="quests dark a" src="https://user-images.githubusercontent.com/889872/103562599-35273e00-4e70-11eb-8624-85ff52add762.png">
<img width="1000" alt="quests dark b" src="https://user-images.githubusercontent.com/889872/103562603-36586b00-4e70-11eb-8cff-976c4ac1e9d3.png">
